Create documentation-government-hyperledger.md

keleffew opened the pull request:
Creating a quickstart guide for hyperledger fabric blockchains in azure government.  Next step is to add img src's.
@smichelotti

azurecla commented:
Hi __@SBArbeit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Mimi approved.

keleffew committed:
Create documentation-government-hyperledger.md

Creating a quickstart guide for hyperledger fabric blockchains in azure government.
@smichelotti

keleffew reviewed (commented):
Here is the base guide without the included <img src's> @smichelotti
